app/web/features/communities/newEvents/EventItem.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Theme,
+  Tooltip,
+  Typography,
+} from "@material-ui/core";
+import { eventImagePlaceholderUrl } from "appConstants";
+import Pill from "components/Pill";
+import dayjs from "dayjs";
+import { useAuthContext } from "features/auth/AuthProvider";
+import { useTranslation } from "i18n";
+import { COMMUNITIES, GLOBAL } from "i18n/namespaces";
+import Link from "next/link";
+import { Event } from "proto/events_pb";
+import { routeToEvent } from "routes";
+import { timestamp2Date } from "utils/date";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  attendees: {
+    display: "flex",
+    alignItems: "flex-end",
+  },
+  card: {
+    display: "flex",
+    width: "100%",
+    height: theme.spacing(20),
+    margin: theme.spacing(2, 0, 2, 0),
+    border: `1px solid ${theme.palette.grey[300]}`,
+    borderRadius: theme.spacing(1),
+  },
+  cardMedia: {
+    height: "100%",
+    width: "25%",
+  },
+  cardContent: {
+    width: "75%",
+    padding: theme.spacing(2, 4),
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+  },
+  eventInfo: {
+    display: "flex",
+    justifyContent: "flex-end",
+    flexDirection: "column",
+  },
+  row: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  tags: {
+    gap: theme.spacing(1),
+  },
+  title: {
+    display: "-webkit-box",
+    boxOrient: "vertical",
+    lineClamp: 2,
+    overflow: "hidden",
+  },
+}));
+
+const EventItem = ({ event }: { event: Event.AsObject }) => {
+  const classes = useStyles({
+    eventImageSrc: event.photoUrl || eventImagePlaceholderUrl,
+  });
+  const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+
+  const startTime = dayjs(timestamp2Date(event.startTime!)).format("llll");
+  const endTime = dayjs(timestamp2Date(event.endTime!));
+
+  const formattedEventDates = `${startTime} - ${endTime.format(
+    endTime.isSame(startTime, "day") ? "LT" : "llll"
+  )}`;
+
+  const {
```

Comment:
since this is in every event item, maybe makes sense to receive it from it's parent? (by prop) [proporsal, not request]

Reply:
IMHO makes more sense for each event to be responsible for its own data and logic. We're already passing the event in the props. It's not something that's being fetched from the DB so just parsing stuff so it's not a heavy calculation.